Sources/JWT/JWT.swift

```diff
@@ -66,6 +76,49 @@ extension Request {
             try self.request.application.jwt.signers.verify(message, as: Payload.self)
         }
 
+        /// Verifies an identity token provided by Apple
+        /// - Parameter identity: The identity token to validate.
+        public func verify(apple identity: String) -> EventLoopFuture<AppleIdentityToken> {
```

Comment:
I think it would be nice to nest this under `req.jwt.apple`. For example, including the previous comment:
```swift
app.jwt.apple.identifier = "..."
app.get(...) { req in
    req.jwt.apple.verify()
}
```

Reply:
I don't really like that.  Why do I have to set the identifier and then in a separate line call verify?  I'd rather just pass the verify in directly.  I'd have to do this on every call anyways.   Making it two statements is just harder on the developer.

Reply:
Many routes may need to fetch the JWT which is what made me think duplicating it everywhere would be annoying. 

```swift
app.jwt.apple.identifier = "..."
app.get("foo") { req in
    req.jwt.apple.verify()
}
app.get("bar") { req in
    req.jwt.apple.verify()
}
app.get("baz") { req in
    req.jwt.apple.verify()
}
```

Reply:
You'd only have one, actually.  You can't use Apple's JWKS for authentication.  It's only good for 9 minutes.  You use it to verify that the initial call really is *you*, and then you return your own authentication/authorization tokens to use thereafter.

Reply:
A bit irritating, but makes sense when you consider you're probably supporting Apple, Google, Facebook, other OAuth, etc...

Reply:
Ah I see. Well if setting the identifier once doesn't seem useful then let's pass it to the verify method. I still think nesting this under `req.jwt.apple` would be nice though. That would allow us to give a proper label to the identifier parameter, too:
```swift
reqp.jwt.apple.verify(identifier: "...")
```